### ISSUE TITLE
Blurry textures after denoising

### ISSUE THREAD

**Issue by PabloDRA** [OpenImageDenoise/oidn#50]
Hi there!
I wanted to comment that I'm having kind of the same problem described in the "Loosing texture details since V1.0".

I've used "Blackman-Harris" pixel filter set up at 1.00 px. 
As you can see after the denoise process the image gets really blurry. The "A" image is when I first noticed it and I thought that it was because the object was quite far from the camera, and there are not too much information/pixels to determine how it should look. 
![A](https://user-images.githubusercontent.com/59416687/71644606-e61d5a80-2ca9-11ea-9921-b18ab58b69da.jpg)

However, at close-ups, it also blurs the image in certain areas while in others don't. 
![Denoiser Luxcore](https://user-images.githubusercontent.com/59416687/71644497-02b89300-2ca8-11ea-9e71-fd69487e7453.png)
![Noisy - Bidir](https://user-images.githubusercontent.com/59416687/71644498-02b89300-2ca8-11ea-9017-cb3d0c3a8996.png)

I don't know if it is something that I might not be set up correctly in my scene or maybe something that the denoiser can't handle yet.

I've been using blender 2.81 and Luxcore for these renders.

Regards, saludos
Pablo

**Comment by jomeng** [comment]
Hi Pablo,

thank you for your report. We are currently working to improve this. Can you provide us with your set of input images by any chance?

**Comment by PabloDRA** [comment]
Hi Jomeng,
Thanks for your reply. Here you will find a link with the free 1024 px textures I used.

https://www.textures.com/download/pbr0145/133180?q=carbon fiber

I can't attach them here.

I've noticed a better behavior of the denoiser after doing some tests with more samples (I'm testing with 2000 samples now) but they are up to a point where it is almost "unnecessary" to use a denoiser.

Regards, saludos
Pablo
